docs/data-sources/space.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_space Data Source - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_space represents a Spacelift space - an inheritable entity holding a collection of customer-defined resources such as stacks, modules, policies, etc. Allows for more granular access control.
```

Comment:
```
an inheritable entity holding a collection of customer-defined resources such as stacks, modules, policies, etc. Allows for more granular access control.
```

a collection of user-defined resources such as stacks, modules, policies, etc. Allows for more granular access control.